validator/client/beacon-api/submit_aggregate_selection_proof.go

```diff
@@ -52,7 +53,12 @@ func (c *beaconApiValidatorClient) submitAggregateSelectionProof(
 		return nil, err
 	}
 
-	aggregatedAttestation, err := convertAttestationToProto(aggregateAttestationResponse.Data)
+	var attData *ethpb.Attestation // Replace with your appropriate struct
```

Comment:
You should unmarshal into the API type, not the protobuf one

Reply:
Not sure how to get it working in with the API type